dev/release/README.md

```diff
@@ -0,0 +1,474 @@
+
+
+    To: general@incubator.apache.org
+    Subject: [RESULT][VOTE] Release Apache SPOT 1.0-incubating
+
+    Hi All, 
+
+    The voting process for the Release Apache SPOT 1.0-incubating is now closed with the following and positive results:
+
+    [3] Binding Votes
+    [1] Non-binding
+
+    Thread of the voting email with responses can be found here:
+
+    https://lists.apache.org/thread.html/32d7c93fe66cc256ed12a5b8f91b57b1d0d659b9012c8f4f13c11191@%3Cgeneral.incubator.apache.org%3E
+
+    
+    I will prepare the artifacts to officially release Apache SPOT 1.0-incubating. 
+
+
+    Thanks
+    --
+    Best Regards!
+    -----------------------------------
+    <Release Manager Name>
+    http://spot.apache.org/
+    -----------------------------------
+
+
+Moving the Artifacts to release stage in SVN using the following command.
+* svn move -m "`<comment>`" `<Directory Origin>` `<Directory Destination>` --username=`<your apache user id>`
+
+Example:
+    
+    svn move -m "Moving Apache SPOT 1.0-incubating release artifacts to release stage" /* https://dist.apache.org/repos/dist/dev/incubator/spot/1.0-incuabting https://dist.apache.org/repos/dist/release/incubator/spot/1.0-incuabting --username=`<your apache user id>`
+
+Allow 24 hours before updating the webpage and announcing the new Release in Apache SPOT (Incubating) webpage. http://nolamarketing.com/client/apache-spot/download/
+
+### **Update WebPages**
+
+You need to update the SPOT webpages to reflect the new release.
+
+### **Announce the release**
+
+Email to the different distribution lists announce@apache.org, user@spot.apache.org, dev@spot.apache.org (using your @apache.org email) For example:
+
+    To: announce@apache.org, user@spot.apache.org, dev@spot.apache.org
+    Subject: [ANNOUNCE] Apache Spot 1.0 (incubating) released
+
+    The Apache SPOT (Incubating) team is pleased to announce the release of SPOT 1.0-incubating.
+
+    This is the first release of SPOT. Major step forward of the project.
+
+    Apache Spot (Incubating) is open source software for leveraging insights from flow and packet analysis. It helps enterprises and service providers gain insight on their network environments through transparency of service delivery and identification of potential security threats or attacks happening among resources operating at cloud scale. While current threat intelligence tools help, identifying unknown threats and attacks remains a challenge.
+
+    The release is available here:
+    http://nolamarketing.com/client/apache-spot/download/ (Update from final version)
+
+    The full change log is available here:
+    https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12320637&version=12340668
+
+    Your help and feedback is more than welcome. For more information on how to report problems and to get involved, visit the project website at http://spot.apache.org/.
+
+    The Apache SPOT (Incubating) Team
```

Comment:
There are some cases where the name of the project is all upper case as _SPOT_. Please replace with camel case as _Spot_ in all the cases.